### ISSUE TITLE
Wrong UDID sent to server when downloading a package

### ISSUE THREAD

**Issue by BLVCKLNGS** [wstyres/Zebra#48]
**Describe the bug**
When the headers are built, you use `[[UIDevice currentDevice] identifierForVendor].UUIString` as a way to get a unique identifier.
This format is not the one that is used "commonly" which causes verifications to fail on repositories end.

**Expected behavior**
I found this bit of code and tested it and it returns the "right" UDID.
```objective-c
# Declare
OBJC_EXTERN CFStringRef MGCopyAnswer(CFStringRef key) WEAK_IMPORT_ATTRIBUTE;

# Use this to get the UDID
CFStringRef UDID = MGCopyAnswer(CFSTR("UniqueDeviceID"));
```

**Comment by wstyres** [comment]
Good to know, I'll fix this for the next beta.

**Comment by wstyres** [comment]
Fixed as of ad9aa0e
